src/app/cli/src/cli_commands.rs

```diff
@@ -437,12 +437,17 @@ pub fn get_command(
             Some(("api-server", server_matches)) => match server_matches.subcommand() {
                 None => {
                     let workspace_svc = cli_catalog.get_one::<WorkspaceService>()?;
+
+                    let current_account_subject = cli_catalog.get_one::<CurrentAccountSubject>()?;
+
                     Box::new(APIServerRunCommand::new(
                         base_catalog.clone(),
                         workspace_svc.is_multi_tenant_workspace(),
                         cli_catalog.get_one()?,
                         server_matches.get_one("address").map(|a| *a),
                         server_matches.get_one("http-port").map(|p| *p),
+                        server_matches.get_flag("get-token"),
+                        current_account_subject,
```

Comment:
It's enough to write `cli_catalog.get_one()?`, the automatic type deduction will do the magic.